Tests/UnitTests/Tests/HttpTests.cpp

```diff
@@ -449,6 +453,71 @@ DEFINE_TEST_CLASS(HttpTests)
         HCHttpCallCloseHandle(call);
         HCCleanup();
     }
+
+    // Temporarily enabled for testing WinHTTP flag changes
+    DEFINE_TEST_CASE(TestGdkHttpRequest)
+    {
+        DEFINE_TEST_CASE_PROPERTIES(TestGdkHttpRequest);
+        
+        // GDK PC integration test to verify HTTP (non-HTTPS) requests work correctly
+        // 
+        // Background: Previously, GDK builds used WINHTTP_FLAG_SECURE_DEFAULTS for all requests,
+        // which could block HTTP requests. The fix ensures HTTP URLs use WINHTTP_FLAG_ASYNC
+        // while HTTPS URLs continue to use WINHTTP_FLAG_SECURE_DEFAULTS.
+        //
+        // This test verifies that HTTP requests can proceed to the network layer without
+        // being blocked by WinHTTP flag configuration issues.
+        
+        VERIFY_ARE_EQUAL(S_OK, HCInitialize(nullptr));
+
+        HCCallHandle call = nullptr;
+        VERIFY_ARE_EQUAL(S_OK, HCHttpCallCreate(&call));
+
+        // Use an HTTP URL (not HTTPS) to test the WinHTTP flags behavior
+        // Using a local/private IP that won't resolve to test flag configuration without external dependency
+        VERIFY_ARE_EQUAL(S_OK, HCHttpCallRequestSetUrl(call, "GET", "http://192.168.1.255/test"));
```

Comment:
don't you have other tests hitting example.com? would that be a way to hit a real http test endpoint and verify SUCCESS rather than hitting a failure and then trying to determine it wasn't an unexpected failure?